### PR TITLE
add linters

### PR DESCRIPTION
**Pull request by santitoot**
it's just a review project

### CODE REVIEW

**Review by ademibowale — COMMENTED** [review]
Hi @santitoot  ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request




Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

You have submitted a project,a student did not make a review at all, you can see only an open pull request

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by bobb-Rob — COMMENTED** [review]
Hi @santitoot, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has not been reviewed by you, kindly ensure that you add at least two review comments using GitHub inline comment feature. Readd the project requirement carefully.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by santitoot** [comment]
@santitoot The HTML code you provided appears to be a basic framework for building a Technical Documentation Page.

This type of project is commonly used to create detailed documentation on a particular technical topic or technology. The page usually contains sections that explain concepts, instructions, code samples, and other relevant details.

In your HTML code, you can see that a basic structure is used with a <head> that includes the page title and a link to an external style sheet file (styles.css). The body (<body>) of the page contains the main content.

The main content is divided into two main sections: the navigation bar (<nav id="navbar">) and the documentation content (<main id="main-doc">).

The navigation bar (<nav>) includes a header that indicates the subject of the documentation ("JS Documentation") and a list of links (<a>) that function as internal links within the page. These links point to different sections within the documentation content using href attributes with values corresponding to the section IDs.

The documentation content (<main>) includes multiple sections (<section>) that represent different topics or concepts to be documented. Each section has a header (<header>) indicating the title of the section, followed by paragraphs (<p>) of text, code examples (<code>), and list items (<li>).

In summary, this project is based on the creation of a JavaScript (JS) technical documentation page, where various concepts, instructions and examples related to JavaScript are explained.

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @santitoot 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Use GitHub Flow

## Required Changes ♻️
_Please kindly check the comments under the review._

- Kindly follow the list from the article ["How to give a code review?"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md).
- What to check 👀
    - If you have used semantic HTML tags 
    - If you applied [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
    - Let's add as many comments as possible, using inline comment allow the author of the project to easily understand the context of your comment 💪

- Watch this [video with an example](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d). 🤗


## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by santitoot** [comment]
Hi @santitoot, 

Good job so far!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct . :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @santitoot , I am @braincee your Code Reviewer

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


so-close

HIGHLIGHTS 🤩 
------

- Great implementation of the role as a code reviewer. 👍 


REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

- [ ] Please as suggested by the previous reviewer, add a minimum of 2 inline comments per the project requirements. This is a link to a [video](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) on how to add inline comments. 😄 

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

- [ ] [OPTIONAL] Please I highly suggest adding Gifs and emojis to make your review more humanly friendly. 😄 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: project.html [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
+<!DOCTYPE html>
+<html lang="en">
 <head>
-  <title>Build a Technical Documentation Page</title>
-  <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
 </head>
```
Comment (santitoot) [inline project.html:8]: nice job fixing your (html) by adding the (!DOCTYPE html) and (html lang="en") to it as well as adding the (meta)

File: style.css [hunk 2]
```diff
@@ -7,6 +7,6 @@
 
 @media only screen and (min-width: 420px) {
   #navbar {
-  position: relative;
+    position: relative;
   }
-}
\ No newline at end of file
+}
```
Comment (santitoot) [inline style.css]: good job fixing this semantic error

**Review by santitoot — COMMENTED** [review]
Hi @santitoot, 

Good job so far!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct . :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by stanleySimeon — APPROVED** [review]
Hi @santitoot, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
